recipes/kalpy/meta.yaml

```
@@ -0,0 +1,65 @@
+{% set name = "kalpy" %}
+{% set version = "0.0.4" %}
+
+package:
+  name: kalpy
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/k/kalpy-kaldi/kalpy-kaldi-{{ version }}.tar.gz
+  sha256: 43cf28d35a97751d4f1b40f133897c67f66dfd81bb0d8d010c826998cca2b49e
+
+build:
+  script:
+    - CXXFLAGS="${CXXFLAGS} -D_LIBCPP_DISABLE_AVAILABILITY" {{ PYTHON }} -m pip install . --no-deps -vv  # [not win]
+  number: 0
+  skip: True  # [py<38]
+  script_env:
+   - KALDI_ROOT={{ environ.get('LIBRARY_PREFIX') }}  # [win]
+   - KALDI_ROOT={{ environ.get('PREFIX') }}  # [not win]
```

I think this is easier - less indirections.
```suggestion
   - KALDI_ROOT=%LIBRARY_PREFIX%  # [win]
   - KALDI_ROOT=$PREFIX           # [unix]
```